spec/controllers/organizers/teams_controller_spec.rb

```diff
@@ -18,7 +18,7 @@
     include_context 'with admin logged in'
 
     describe 'GET index' do
-      let!(:sponsored_team) { create :team, :in_current_season, kind: 'sponsored' }
+      let!(:sponsored_team) { create :team, :in_current_season, kind: 'full_time' }
```

Comment:
I'd suggest to also rename this to `full_time`throughout the tests, just to make the intend more obvious _(we probably now would consider both sort of teams "sponsored")_